materialize-snowflake/config.go

```diff
@@ -42,24 +40,63 @@ type advancedConfig struct {
 	UpdateDelay string `json:"updateDelay,omitempty" jsonschema:"title=Update Delay,description=Potentially reduce active warehouse time by increasing the delay between updates. Defaults to 30 minutes if unset.,enum=0s,enum=15m,enum=30m,enum=1h,enum=2h,enum=4h"`
 }
 
-// ToURI converts the Config to a DSN string.
-func (c *config) ToURI(tenant string) string {
-	// Build a DSN connection string.
-	var configCopy = c.asSnowflakeConfig(tenant)
+// toURI manually builds the DSN connection string
```

Comment:
Nit: Comments should usually be complete sentences, including punctuation at the end. That's clearly not a universally adhered to rule but I'd suggest working in that direction, which will require your new comments to have periods at the end.

Reply:
I made most of my new comments be complete sentences with punctuation.